pkg/arvo/sys/vane/ames.hoon

```diff
@@ -1420,6 +1424,17 @@
     |=  [=wire error=(unit tang)]
     ^+  event-core
     ::
+    ?:  ?=([%alien @ ~] wire)
```

Comment:
Unless I'm missing something, as written, this will keep re-sending the sendkeys packet forever.  To end it, we should check whether our ship-state for this peer has gone from %alien to %known, and only re-send if not.

Reply:
good catch, fixed